.github/workflows/azure_automation_test.yml

```diff
@@ -289,21 +298,34 @@ jobs:
             }'
           wait-for-completion: false
 
-  notify-slack-on-failure:
+  # The first step here is to notify ia-notification-test on failure from either scheduled runs or staging hook
+  # The second step here is to notify dsde-qa on runs from the staging beehive hook
+  notify-slack:
     runs-on: ubuntu-latest
     needs:
       - init-github-context
       - create-bee-workflow
       - leo-automation-test-job
       - destroy-bee-workflow
-    if: ${{ github.event_name == 'schedule' && failure() }} # Want to notify regardless of which step fails
     steps:
-      - name: Notify slack
-        uses: slackapi/slack-github-action@v1.23.0
+      - id: notify-ia-on-failures
+        name: Notify ia on failures
+        if: ${{ failure() && github.ref_name == 'develop' }} # Want to notify regardless of which step fails.
```

Comment:
because you have PRs enabled on develop for this - is this what you are wanting? 
from the documentation:
> For pull requests, the format is <pr_number>/merge.

Reply:
I believe as is, this will only run on committing to develop (and now that I think of it, not really sure what pull-request with a branch of develop would actually do, outside of maybe PRs from develop?) - thinking out loud

Reply:
So there are to relevant sections in the `on` block, one is for `pull_request` (i.e. on every commit for PRs), on is `push`. This block should only trigger when the branch it is running against is develop. In other words, every time we merge/commit something to the develop branch itself we get an additional test run and the report to our channel. The idea here is, when the suite is actually stable, anytime it fails against the develop branch its a red flag.

```
  pull_request:
    branches: [ develop ]
    paths-ignore: [ '**.md' ]
  push:
    branches: [ develop ]
    paths-ignore: [ '**.md' ]
```